### PR TITLE
Removed printStackTrace(...) from whole code, issue #660

### PR DESCRIPTION
**Pull request by alexkravin**
According to #660 

Removed printStackTrace() from whole code.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/Main.java [hunk 1]
```diff
@@ -153,7 +152,6 @@ private static Checker createChecker(Configuration config,
         catch (final Exception e) {
             System.out.println("Unable to create Checker: "
                                + e.getMessage());
-            e.printStackTrace(System.out);
```
Comment (mkordas) [inline src/main/java/com/puppycrawl/tools/checkstyle/Main.java]: Shouldn't we replace it somehow? Stacktraces may be useful for users and us.

Comment (alexkravin) [inline src/main/java/com/puppycrawl/tools/checkstyle/Main.java]: See the opened issue.
In this example message's text will be shown in any case

**Comment by romani** [comment]
let no mix all printStacktrace usages together.

Please revert changes at DefaultLogger.java as it prints exception to result stream.

We need to focus on printStacktrace that do print to system output during non recoverable Exception.

If Exception is recoverable all should be reported to Loggers and, checkstyle should not put anything to system out. System out could not exists at all, or nobody listen to it (eclipse plugin)

**Comment by romani** [comment]
Please also remove changes at XmlLogger, logger know how to put exception in result stream.

**Comment by alexkravin** [comment]
Updated

**Comment by romani** [comment]
merged as FF

**Pull request closed without merging**